server/jetty/README.md

```diff
@@ -1,5 +1,44 @@
 # Native packaging for Deephaven Jetty server
 
+### Setting up a Python virtual environment
+
+This is an optional prerequisite, but lets the later commands to run the server work without
+specifying groovy as the console language. If you skip this step, be sure to use groovy, or
+there will be errors on startup indicating that the Python environment is not suitable.
+
+See https://github.com/deephaven/deephaven-core/issues/1657 for more discussion
+
+1. On MacOS there is an extra patch to apply at this time, to add an extra argument to clang,
+see the first code snippet of the comment at
+https://github.com/deephaven/deephaven-core/issues/1657#issuecomment-989040798 for specifics.
+1. Make sure Python is installed with a shared library. For example, if using `pyenv install`,
+be sure to first set `PYTHON_CONFIGURE_OPTS="--enabled-shared"`.
+1. Make a new directory for a virtual environment, and set it up:
```

Comment:
Many `1.`s.

Reply:
This renders cleanly in markdown - using 1. each time ensures you don't rely on the text to be correct, can easily insert numbers, but the rendered html will be numbered correctly.